src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,10 +25,8 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(
-            ~TargetFrameworkMonikers.Netcoreapp, 
-            "NetFramework: NetEventSource is only part of .NET Core;" + 
-            "UapAot: RemoteExecutorConsoleApp.exe crashes trying to load netstandard.dll - StrongName issue")]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/19909", TargetFrameworkMonikers.Uap)]
```

Comment:
I saw others using this pattern. 
I think it's interesting to have the full URI in case the bugs causing the test to fail are in other repos (like coreclr or buildtools, etc).